### PR TITLE
Corrected check for non-existence of volume after delete

### PR DESCRIPTION
**Pull request by shtripat**
Also added enviornment variable `ANSIBLE_HOST_KEY_CHECKING=False`
for making sure first time gdeploy execution doesnt ask for
confirmation to connect to node.

Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@r0h4n @nnDarshan plz review

File: tendrl/gluster_integration/objects/volume/atoms/delete/__init__.py [hunk 1]
```diff
@@ -83,13 +86,30 @@ def run(self):
...
+                ),
+                recursive=True
+            )
+        except etcd.EtcdKeyNotFound as ex:
+            Event(
+                ExceptionMessage(
+                    priority="debug",
+                    publisher=NS.publisher_id,
+                    payload={
+                        "message": "Given key is not present in etcd.",
```
Comment (r0h4n) [inline tendrl/gluster_integration/objects/volume/atoms/delete/__init__.py]: what kinda msg is this? 

Id prefer  "Cannot delete volume %vol_id, volume does not exist "

File: tendrl/gluster_integration/objects/volume/atoms/delete/__init__.py [hunk 2]
```diff
@@ -83,13 +86,30 @@ def run(self):
...
-            "clusters/%s/Volumes/%s" % (
-                NS.tendrl_context.integration_id,
-                self.parameters['Volume.vol_id']
-            ),
-            recursive=True
-        )
+        # Intentionally keeping sleep with high value and blocking
+        # one and not the gevent.sleep(). Make sure no sync etc
+        # happens for this time and then execute etcd delete for volume
+        time.sleep(30)
```
Comment (r0h4n) [inline tendrl/gluster_integration/objects/volume/atoms/delete/__init__.py]: put a check in NS.gdeploy_plugin.delete_volume to block till volume is actually deleted in sds, 

time.sleep doesnt belong here

Comment (shtripat) [inline tendrl/gluster_integration/objects/volume/atoms/delete/__init__.py]: Problem here is that the volume gets deleted from gluster cluster but its not deleted from etcd. The gdeploy plugin has returned a success after deletion and then the below etcd delete key was success.

If we used to put a pdb to debug it always used to work, but in normal flow it never used to delete the key and so post check to verify if the volume is deleted successfully used to fail.

So actually waiting inside gdeploy plugin is not the place this is required. Problem is while deletion from etcd.

This explicit time.sleep() helped so kept it like this.

File: tendrl/gluster_integration/objects/volume/atoms/delete/__init__.py (hunk 1 above)
Comment (shtripat) [inline tendrl/gluster_integration/objects/volume/atoms/delete/__init__.py]: This is copy + paste issue :) Will correct the message.

File: tendrl/gluster_integration/objects/volume/atoms/delete/__init__.py (hunk 2 above)
Comment (shtripat) [inline tendrl/gluster_integration/objects/volume/atoms/delete/__init__.py]: I would add a while loop and wait for the deletion to happen here may be.

**Pull request merged**